budget-binder-multiplatform-app/src/jsMain/kotlin/de/hsfl/budgetBinder/common/StateManager.kt

```diff
@@ -6,7 +6,8 @@ import de.hsfl.budgetBinder.common.Constants.BASE_URL
 import de.hsfl.budgetBinder.presentation.Screen
 
 object StateManager {
-    val screenState = mutableStateOf<Screen>(Screen.Welcome)
+    // TODO: screenState replaced with new Router
```

Comment:
Schreib bitte noch etwas zu der Strukturänderung in die Docs! Da fehlt bis jetzt noch ein abstract drüber

Reply:
StateManager kommt weg!

Reply:
Und das gehört dokumentiert! "Wir hatten ein StateManager, der [...] konnte, haben ihn aber ausgetauscht, da [...]"

Reply:
Aber das war noch für den HiFi, damit die Daten wie z.B. User Informationen Global gespeichert werden können... natürlich kann ich das Dokumentieren...

Reply:
Gut so! Auch das mit dem HiFi gehört dokumentiert! Alles was wir haben, sollten wir auch aufschreiben , natürlich strukturiert.